mantidproject/mantid
# Main Cleanup of Diffraction release notes

Reordered, notes amalgamated where appropriate, spelling & grammar check, screenshot added.

**Description of work.**
The main refactoring the the Diffraction release notes ahead of release of 6.2. Includes:-

- Amalgamating notes where appropriate
- Adding/fixing links
- Re-ordering information
- Spelling & Grammar check
- Adding a screenshot


**To test:**
- remove ```build/docs/doctrees```
- build the documents again and check there are no warnings
- Check that the links work
- Check the information is still correct
- Are the images visible and clearly illustrate the new feature/improvement?
- Grammar/Spelling check

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
